src/commands/importExportEntities/entities/webhooks.ts

```diff
@@ -0,0 +1,44 @@
+import { WebhookContracts } from "@kontent-ai/management-sdk";
+import chalk from "chalk";
+
+import { logInfo } from "../../../log.js";
+import { serially } from "../../../utils/requests.js";
+import { FixReferences } from "../../../utils/types.js";
+import { EntityDefinition } from "../entityDefinition.js";
+
+type Webhook = FixReferences<WebhookContracts.IWebhookContract>;
+
+export const webhookEntity: EntityDefinition<ReadonlyArray<Webhook>> = {
```

Comment:
you have listed this entity neither in the export command nor the import command -> it won't be called

Reply:
added